projects/docs/docs/commands/graph.md

```diff
@@ -15,6 +15,7 @@ saying goes, "one image is worth a thousand words":
 The command will output the dependency graph as an image, in the `png` format.
 You can also change the format to `dot` (see [DOT](<https://en.wikipedia.org/wiki/DOT_(graph_description_language)>))
 to get the raw contents of the graph.
+You can also change the format to `json` to export the project structure into a JSON file.
```

Comment:
Because we are exporting a JSON, I anticipate users building workflows upon that. That means the output becomes a public interface between the users and the CLI. Until the Graph's structure stabilizes, I'd add an [admonition](https://docusaurus.io/docs/markdown-features/admonitions) here making it clear that the format is subject to breaking changes.

Reply:
Sounds good!